backend/app/routes/email.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, Depends
+from app.services.interfaces.email_service import IEmailService
+from app.services.email.email_service import EmailService
+from app.services.email.email_service_provider import AmazonSESEmailProvider
+
+router = APIRouter()
+
+def get_email_service() -> IEmailService:
+    email_provider = AmazonSESEmailProvider(aws_access_key = "", aws_secret_key ="")
+    return EmailService(email_provider)
+
+@router.post("/send-test-email/")
+async def send_welcome_email(recipient: str, user_name: str, email_service: IEmailService = Depends(get_email_service)):
```

Comment:
```suggestion
async def send_welcome_email(recipient: str, user_name: str, email_service: Annotated[IEmailService, Depends(get_email_service)]):
``` 

I think it's recommended to use [Annotated](https://fastapi.tiangolo.com/tutorial/dependencies/#share-annotated-dependencies), at least for code deduplication it makes sense. Don't need to spend too much time if it doesn't work though.